core/src/main/java/io/questdb/cutlass/pgwire/BindVariableSetter.java

```diff
@@ -27,6 +27,6 @@
 import io.questdb.griffin.SqlException;
 
 @FunctionalInterface
-interface BindVariableSetter {
+public interface BindVariableSetter {
```

Comment:
Do you think this change could fix the issue we are having with bound parameters and PGWire?